docs/Example Schedules/CE Schedules/compe-kteal2.md

```diff
@@ -0,0 +1,185 @@
+# Computer Engineering, Statistics Minor, 7-8 Semesters
+
+Hi, I'm Keegan, currently a second-semester Junior (CE '25) and I'm pursuing a Statistics minor. I'm focused on software, robotics, autonomy, and controls. Originally, I had planned to use one semester as a co-op but decided to take more technical electives instead. My schedule is definitely on the heavy side but it allows you to spend most of Junior and Senior year taking tech electives that you're interested in, while never having to double up on the big 4 ECE classes.
+
+## Features of My Schedule
+
+- Can be easily modified to 7 semesters (or less)
+- No summer or winter classes, so internship options every summer
+- Statistics minor + James Scholar
+
+## Schedule
+
+*Difficulty rating is subjective to the time I took the courses, so YMMV*
+
+### AP Credits and Proficiency Tests
+| Course Name | Credit Hours |
+| --- | --- |
+| CHEM 102 | 3 |
+| CHEM 104 | 3 |
+| CS 101 | 3 |
+| ECON 102 | 3 |
+| ECON 103 | 3 |
+| IB 150 | 4 |
+| MATH 220 | 5 |
+| MATH 231 | 3 |
+| MCB 150 | 4 |
+| PHYS 211 | 4 |
+| PS 101 | 3 |
+| RHET 105 | 4 |
+| &nbsp; | &nbsp; |
```

Comment:
don't add this, instead bold "total" and "42".  Do the same for all of the other tables.